npc/kafras/functions_kafras.txt

```diff
@@ -189,6 +259,18 @@ function	script	F_Kafra	{
 			callfunc "F_KafStor",1,0;
 			next;
 		}
+		else if (.@K_Menu0$[.@j] == "Use Storage 1") {
+			callfunc "F_Storage",4,0;
+			next;
+		}
+		else if (.@K_Menu0$[.@j] == "Use Storage 2") {
+			callfunc "F_Storage",1,5;
+			next;
+		}
+		else if (.@K_Menu0$[.@j] == "Use Storage 3") {
+			callfunc "F_Storage",2,6;
```

Comment:
Function not found! [F_Storage]

Reply:
This function is a part of #8682, it is not merged it!